edx_name_affirmation/tasks.py

```diff
@@ -37,7 +37,6 @@ def idv_update_verified_name(self, attempt_id, user_id, status, photo_id_name, f
                 'status': status
              }
              )
-    trigger_status = VerifiedNameStatus.trigger_state_change_from_idv(status)
```

Comment:
Ah... you will have a temporary problem if mixed statuses during deployment. If the website code gets deployed first, then the celery code would be confused by the statuses passed in. If the celery code deploys first, some of the outstanding queued celery tasks would have wrong statuses input. I say don't change this here.